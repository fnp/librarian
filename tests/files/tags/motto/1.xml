<autor_utworu>Aleksander Fredro</autor_utworu>
<nazwa_utworu>Zemsta</nazwa_utworu>

<motto>
<akap>Nie masz nic tak złego, żeby się na dobre nie przydało. Bywa z węża dryjakiew, złe często dobremu okazyją daje.</akap>
</motto>

<motto_podpis>
And. Maks. Fredro
</motto_podpis>
