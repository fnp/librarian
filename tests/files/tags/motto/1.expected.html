<div id="book-text">
<h1>
<span class="wl author" id="i1">Aleksander Fredro</span><span class="wl title" id="i2">Zemsta</span>
</h1>
<div class="motto" id="i3">
<a href="#f1" class="wl-num">1</a><p class="wl paragraph" id="f1">Nie masz nic tak złego, żeby się na dobre nie przydało. Bywa z węża dryjakiew, złe często dobremu okazyją daje.</p>
</div>
<p class="wl motto_podpis" id="i4">
And. Maks. Fredro
</p>
</div>
